crates/aptos/src/move_tool/mod.rs

```diff
@@ -222,6 +227,33 @@ impl CliCommand<&'static str> for TestPackage {
     }
 }
 
+#[async_trait]
+impl CliCommand<&'static str> for TransactionalTestOpts {
+    fn command_name(&self) -> &'static str {
+        "TransactionalTest"
+    }
+
+    async fn execute(self) -> CliTypedResult<&'static str> {
+        let root_path = self
+            .root_path
+            .clone()
+            .into_os_string()
+            .into_string()
+            .map_err(|_| CliError::CommandArgumentError("--root_path".to_string()))?;
```

Comment:
This is probably cleaner:
`let root_path = self.root_path.display().to_string();`